client/extensions/woocommerce/style.scss

```diff
@@ -12,8 +12,18 @@
 	@import 'components/form-dimensions-input/style';
 }
 
-.store-sidebar__sidebar {
+@keyframes slideInFromLeft {
+	0% {
+		transform: translateX(-273px);
```

Comment:
There needs to be [spacing around parenthesis](https://github.com/Automattic/wp-calypso/blob/de700a238a201232c8481b59b4f7feba7700e977/docs/coding-guidelines/css.md#spacing).